README.md

```diff
@@ -8,19 +8,21 @@
 
 ## Tools Used
 
-> Add a list of tools used like frameworks, libraries, task runners, test runners, bundlers, linters, template engines, etc.
+> Add a list of tools used like frameworks, libraries, task runners, test runners, bundlers, linters, template engines, etc.1a
 
-
->Framework
-[ReactJS] - [Source](https://reactjs.org) 
 >Library
-[D3.jS]   - [Source](https://d3js.org)
+
+1. ReactJS - [Source](https://reactjs.org)
+2. D3.jS   - [Source](https://d3js.org)
 
 
 ## Install and Build
 
 > Describe how to install, build, and test your app
 
+to  install necessary tools run "npm install" script in root directory root directory --> (cs01-team-edsger-04/)
```

Comment:
You could just copy-paste the content of the other `README` here instead of this sentence.